test/auth-modes.test.js

```diff
@@ -0,0 +1,1139 @@
+                                    getTokenTransferMethod: () => conf.getTokenTransferMethodRes,
+                                    antiCsrf: "VIA_TOKEN",
+                                }),
+                            ],
+                        });
+
+                        const app = getTestApp();
+
+                        const createInfo = await createSession(app, "cookie");
+
+                        const authMode =
+                            conf.authCookie && conf.authHeader
+                                ? "both"
+                                : conf.authCookie
+                                ? "cookie"
+                                : conf.authHeader
+                                ? "header"
+                                : "none";
+
+                        const res = await testGet(
+                            app,
+                            createInfo,
+                            conf.sessionRequired ? "/verify" : "/verify-optional",
+                            conf.output === "UNAUTHORISED" ? 401 : 200,
+                            authMode
+                        );
+                        switch (conf.output) {
+                            case "undefined":
+                                assert.strictEqual(res.body.sessionExists, false);
+                                break;
+                            case "UNAUTHORISED":
+                                assert.deepStrictEqual(res.body, { message: "unauthorised" });
+                                break;
+                            case "validatecookie":
+                            case "validateheader":
+                                assert.strictEqual(res.body.sessionExists, true);
+                        }
+                    });
+
+                    it(`should match line ${i + 1} with a expired token`, async () => {
+                        await setKeyValueInConfig("access_token_validity", 2);
+                        await startST();
+                        SuperTokens.init({
+                            supertokens: {
+                                connectionURI: "http://localhost:8080",
+                            },
+                            appInfo: {
+                                apiDomain: "api.supertokens.io",
+                                appName: "SuperTokens",
+                                websiteDomain: "supertokens.io",
+                            },
+                            recipeList: [
+                                Session.init({
+                                    getTokenTransferMethod: () => conf.getTokenTransferMethodRes,
+                                    antiCsrf: "VIA_TOKEN",
+                                }),
+                            ],
+                        });
+
+                        const app = getTestApp();
+
+                        const createInfo = await createSession(app, "cookie");
+                        await delay(3);
```

Comment:
Should be avoided as it will keep on slowing tests.

Reply:
Sure, but AFAIK, we do not have a better way of making a token expire.

Reply:
How about updating token content to make it expired.